### DISCUSSION TITLE
Taxonomy Term Shared and Local properties

### DISCUSSION THREAD

**Discussion by JoHoN8** [pnp/pnpcore#1005] [General]
Is it possible to retrieve the Shared and/or Local properties of a term. 
I have searched and have not been able to find anything. 
If not will this functionality been added at some point?

**Comment by jansenbe** [comment]
Hi @JoHoN8 ,

The implementation in PnP Core SDK depends on the Microsoft Graph Taxonomy APIs and they do not yet support local properties. So today only shared properties can be read/updated/added/deleted.

Is there a reason why you specifically require local properties?

**Comment by JoHoN8** [comment]
Values are stored on the term as local properties that why I need to access them. 
I kind of figured the answer would be NO, so I wrote a powershell application to read the local properties and create a sharepoint list that I can reference. 
Thanks any way!
